sentinelhub/data_utils.py

```diff
@@ -3,17 +3,18 @@
 """
 from typing import Any, Dict, List, Optional, Tuple, Union
 
-from .exceptions import PandasNotInstalled
 from .time_utils import parse_time
 from .type_utils import JsonDict
 
+_PANDAS_IMPORT_MESSAGE = "The Pandas backend is not installed by default. We suggest you install the `pandas` package."
```

Comment:
Change to something along the lines of `"To use this function you need to install the `pandas` library, which is not a requirement of `sentinelhub-py`." The docstring of the functions should also mention that